docs/Manage-Space.rst

```diff
@@ -293,28 +298,64 @@ One of the two following cases occurs when you select this tab.
 
 |image27|
 
+.. _SpaceHomePortlets:
+
+**Space home portlets**
+
+Three portlets are available for space homepage:
+
+-  Description: should display the space's description and the list of the space's managers.
+-  :ref:`Calendar <CalendarApp>`: displays the today's events that are added to the space's calendar.
+-  :ref:`Who is on line? <WhoIsOnlineApp>`: displays the on line members of the space.
+
+.. note:: :ref:`Calendar <CalendarApp>` and :ref:`Who is on line? <WhoIsOnlineApp>` portlets are the same
+          as for :ref:`intranet homepage <PLFHomepage>` but specific for the space's calendar and members.
+
+By default, only two portlets appear in the newly created space: **Description** and **Calendar**.
```

Comment:
This sounds redundant with the previous paragraph.

Reply:
In the previous one I presented the available portlets for the space home page in general, here I present the defaults ones i.e. those which appears just after space's creation (the Who is online appears only when a member accepts the invitation and that he is online)

Reply:
I am not sure to understand what `the available portlets for the space home page in general` means. We can add any portlets in the space home page by editing the page, not only those you listed.

Reply:
hmmm OK when I read the spec https://community.exoplatform.com/portal/intranet/wiki/group/spaces/platform_4/Portlet_in_Space_Homepage_(MVP) i thought that only these three portlet could be added to a space

Reply:
No, nothing prevents the admin from editing the page and adding/deleting portlets.